distributionviewer/core/static/js/app/components/views/chart-menu.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+
+export class ChartMenu extends React.Component {
+  render() {
+    return (
+      <nav className="filters">
+        <ul>
+          {this.props.items.map(metric => {
+            return (
+              <li key={`menu-${metric.name}`}>
+                <a href={`/metric/${metric.name}/`}>{metric.name}</a>
```

Comment:
Does React have any concept of a reverse URL resolver so you can define your URLs in one place?